.eleventy.js

```diff
@@ -2,6 +2,12 @@ const yaml = require("js-yaml");
 
 module.exports = function (eleventyConfig) {
   eleventyConfig.addDataExtension("yaml", (contents) => yaml.load(contents));
+  eleventyConfig.addPassthroughCopy(
```

Comment:
Cant, we remove `serve` with this change now?
Everything should not work with `eleventy --serve --watch` directly right?

Reply:
Oh, wasn't aware of the `--serve` option. Yeah I think that should work, let me give this a try.

Reply:
Works :+1: